### PR TITLE
CV2-4126 shift to use video via sync endpoint

### PR DESCRIPTION
**Pull request by DGaffney**
## Description

Shifts video to use the new sync endpoints in lieu of legacy code path

References: 4126

## How has this been tested?

No tests yet - will require extensive testing against alegre and in integration tests

## Things to pay attention to during code review

Does it even do video matching? Will need to confirm other modalities as well during integration tests.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [ ] I have added regression tests, if the PR fixes a bug
- [ ] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

### CODE REVIEW

**Comment by caiosba** [comment]
I'll review once the whole stack is working.

File: app/models/bot/alegre.rb [hunk 1]
```diff
@@ -151,7 +151,7 @@ def self.run(body)
       if body.dig(:event) == 'create_project_media' && !pm.nil?
         Rails.logger.info("[Alegre Bot] [ProjectMedia ##{pm.id}] This item was just created, processing...")
         self.get_language(pm)
-        if self.get_pm_type(pm) == "audio" || self.get_pm_type(pm) == "image"
+        if self.get_pm_type(pm) == "audio" || self.get_pm_type(pm) == "image" || self.get_pm_type(pm) == "video"
```
Comment (melsawy) [inline app/models/bot/alegre.rb:154]: For simplicity you can use this one
`['audio', 'image', 'video'].include?(self.get_pm_type(pm))`

**Review by melsawy — APPROVED** [review]
Look good to me just leave one comment for simplicity

**Review by caiosba — APPROVED** [review]
Tested locally.

**Pull request merged**